src/spring/azext_spring/_params.py

```diff
@@ -267,6 +267,8 @@ def load_arguments(self, _):
                    help='A json file path for the persistent storages to be mounted to the app')
         c.argument('loaded_public_certificate_file', options_list=['--loaded-public-certificate-file', '-f'], type=str,
                    help='A json file path indicates the certificates which would be loaded to app')
+        c.argument('deployment_name', type=str, default='default',
+                   help='Name of the default deployment.', validator=validate_name)
```

Comment:
`str` is the default type, so it does not need to be defined repeatedly
```suggestion
        c.argument('deployment_name', default='default',
                   help='Name of the default deployment.', validator=validate_name)
```

Reply:
Thanks for your suggestion. Will do